test/extended/builds/jenkins_autoprovision.go

```diff
@@ -0,0 +1,58 @@
+package builds
+
+import (
+	"fmt"
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[Feature:Builds][Slow] jenkins autoprovision", func() {
+	defer g.GinkgoRecover()
+	var (
+		envVarsPipelinePath = exutil.FixturePath("testdata", "samplepipeline-withenvs.yaml")
+		oc                  = exutil.NewCLI("jenkins-pipeline", exutil.KubeConfigPath())
+	)
+
+	g.Context("Pipeline should build and complete successfully", func() {
+
+		g.BeforeEach(func() {
+			exutil.DumpDockerInfo()
+
+			g.By("waiting for default service account")
+			err := exutil.WaitForServiceAccount(oc.KubeClient().Core().ServiceAccounts(oc.Namespace()), "default")
+			o.Expect(err).NotTo(o.HaveOccurred())
+			g.By("waiting for builder service account")
+			err = exutil.WaitForServiceAccount(oc.KubeClient().Core().ServiceAccounts(oc.Namespace()), "builder")
+			o.Expect(err).NotTo(o.HaveOccurred())
+		})
+
+		g.AfterEach(func() {
+			if g.CurrentGinkgoTestDescription().Failed {
+				exutil.DumpPodStates(oc)
+				exutil.DumpPodLogsStartingWith("", oc)
+			}
+		})
+
+		g.It("where jenkins is autoprovisioned by the admission controller", func() {
+
+			// instantiate the bc
+			g.By(fmt.Sprintf("calling oc new-app -f %q", envVarsPipelinePath))
+			err := oc.Run("new-app").Args("-f", envVarsPipelinePath).Execute()
+			o.Expect(err).NotTo(o.HaveOccurred())
+
+			// start the build
+			g.By("starting the pipeline build, including env var, and waiting for it to complete")
+			br, err := exutil.StartBuildAndWait(oc, "-e", "FOO2=BAR2", "sample-pipeline-withenvs")
```

Comment:
we don't need to run anything, it should be sufficient to confirm the jenkins pod gets created.  (don't even really need to wait for it to start up)

Reply:
update pushed